tests/test_handlers.py

```diff
@@ -434,3 +434,21 @@ async def test_folder_creation_works(self):
         self.MockedDbService().create.assert_called_once()
         self.assertEqual(response.status, 201)
         self.assertEqual(json_resp['folderId'], self.folder_id)
+
+    @unittest_run_loop
+    async def test_get_folders(self):
+        """Test get_folders() endpoint returns empty list."""
+        self.MockedDbService().db_client = MagicMock()
```

Comment:
this doesn't actually test much. If you want to test get_folders, I'd suggest you to mock `db_service.query` to return avaitable cursor, probably with the help from MockCursor class in test_operators.

Reply:
this was also very helpful 👍